src/components/Button/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import {Button} from 'react-native';
+
+CustomButton = ({value, style, color, onPress}) => {
+  return (
+    <Button
+      title={value}
+      style={style}
+      color={color}
```

Comment:
is "color" property not included in the style itself. If no, then it's fine but if yes, try to use the only color inside style property itself.